src/Symfony/Bridge/PhpUnit/Legacy/SetUpTearDownTraitForV5.php

```diff
@@ -30,13 +32,17 @@ public static function setUpBeforeClass()
     public static function tearDownAfterClass()
     {
         self::doTearDownAfterClass();
+
+        parent::doTearDownAfterClass();
```

Comment:
not sure: it's the responsibility of self::doTearDownAfterClass (reimplemened in the using class) to call parent::tearDownAfterClass instead

Reply:
For one, I messed up the parent calls in a couple of places by calling the `do...` methods.

Also, after playing around with different setups, I'm not sure this is needed after all; I may just have done something wrong. Let me re-investigate and report back.

Reply:
I think this PR is not needed yes, that's the point of my comment :)